### PR TITLE
feat: add support for planet selection via CLI

### PR DESCRIPTION
**Pull request by NicholasCostanzo**
Adds feature to pass `-p <planet_id>` or `--planet <planet_id>` in CLI.
Script will attempt to use the planet if its available, otherwise the default behavior will be used.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -586,22 +587,34 @@ class SalienScript {
...
-          const planet = this.knownPlanets.get(planetId);
+      const selectedPlanet = this.knownPlanets.get(this.planet);
 
-          if (this.skippedPlanets.includes(planetId) || !planet[priority[i]]) {
-            return;
-          }
+      if (!selectedPlanet || this.skippedPlanets.includes(this.planet)) {
+        logger(
+          this.name,
+          `>> Selected planet ${chalk.yellow(this.planet)} not available. Slecting next available planet...`,
```
Comment (resir014) [inline src/index.js]: Planet id here shows up on the log as `null` if no `--planet` flag is set. Consider setting up a guard to check if we actually set a `--planet`

Also, typo: `Slecting` -> `Selecting`

image

**Review by resir014 — CHANGES_REQUESTED** [review]
Thank you so much for this, you're a legend! ❤️ 

I've run your branch as a test and I've found one minor issue. Once fixed, this is good to merge!

**Review by resir014 — APPROVED** [review]
LGTM, thanks for this!

**Comment by NicholasCostanzo** [comment]
Happy to help :)

**Comment by resir014** [comment]
Hey @NicholasCostanzo, sorry but we'll have to revert this PR for now since some users have reported issues with it.

We're still investigating, but we might be able to bring this back after [our rewrite](https://github.com/South-Paw/salien-script-js/pull/19) is complete.

Thanks for your help though!

**Pull request merged**